dashboard/app/models/user.rb

```diff
@@ -2587,4 +2592,41 @@ def code_review_groups
     errors.add(:parent_email) unless parent_email.nil? ||
       Cdo::EmailValidator.email_address?(parent_email)
   end
+
+  US_STATE_DROPDOWN_OPTIONS = {
```

Comment:
weird corner case - what about US Territories? Guam, the Virgin Islands, American Samoa, etc.

Reply:
I chatted with product and legal, we are okay with just the 50 states and Washington DC. I also ran some quick experiments with our geocoder, and it looks like it identifies those territories as their own countries. Students in those territories shouldn't see this UI because we only show it IP's in the United States IP range.